### PR TITLE
Tranforming to Cloud Native Finland page

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -1,18 +1,42 @@
...
         </div>
     </div>
     <div class="col-lg-6">
         <img src="/images/kcd-logo-color.svg" alt="KCD logo" style="max-width: 300px;">
     </div>
 </div>
 
+# Mission
+
+Our mission is to promite cloud native technologies in Finland by sharing knowledge and create space for technologists
```
Comment (LinAnt) [inline content/en/_index.md:20]:
```suggestion
Our mission is to promote cloud native technologies in Finland by sharing knowledge and create space for technologists
```

File: content/en/_index.md [hunk 2]
```diff
@@ -1,18 +1,42 @@
...
+to shair their experiences and ideas.
```
Comment (LinAnt) [inline content/en/_index.md:21]:
```suggestion
to share their experiences and ideas.
```

**Review by LinAnt — APPROVED** [review]

File: content/en/_index.md (hunk 1 above)
Comment (MaryamTavakkoli) [inline content/en/_index.md:20]:
```suggestion
Our mission is to promite cloud native technologies in Finland by sharing knowledge and creating space for technologists
```

File: content/en/_index.md [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
+
+# What we do?
+
+## Meetups 
+
+We are organizing the [Kubernetes and CNCF Finland Meetup](https://www.meetup.com/Kubernetes-Finland/)
+
+![](https://secure.meetupstatic.com/photos/event/a/2/1/8/clean_485021496.webp)
+
+## Kubernetes Community Day (KCD)
```
Comment (MaryamTavakkoli) [inline content/en/_index.md:31]:
```suggestion
## Kubernetes Community Days (KCD)
```

File: content/en/contact/_index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  main:
+    weight: 30
+---
+
+## Contact us
+
+
+## Social media
+
+* Follow us on LinkedIn.
```
Comment (MaryamTavakkoli) [inline content/en/contact/_index.md:16]: We can give a link https://www.linkedin.com/company/kcd-helsinki

File: content/en/events/_index.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+linkTitle: Events
+toc_hide: false
+list_pages: false
+menu:
+  main:
+    weight: 20
+description: Events related to Cloud Native Finland
+---
+
+## Kubernetes Comminity Day Helsinki 2025
```
Comment (MaryamTavakkoli) [inline content/en/events/_index.md:12]:
```suggestion
## Kubernetes Community Day Helsinki 2025
```

File: content/en/events/_index.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community (us) organized event
```
Comment (MaryamTavakkoli) [inline content/en/events/_index.md:14]:
```suggestion
We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community-organized event
```

File: content/en/events/_index.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+colects adopters and technologists from open source and cloud native communities to share experiences and ideas. This
```
Comment (MaryamTavakkoli) [inline content/en/events/_index.md:15]:
```suggestion
brings together adopters and technologists from open source and cloud native communities to share experiences and ideas. This
```

File: content/en/events/_index.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+event is supported by the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io)
```
Comment (MaryamTavakkoli) [inline content/en/events/_index.md:16]:
```suggestion
event is supported by the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io).
```

File: content/en/events/_index.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+### Sponsors
+
+We are looking for sponsors!
+
+## Kubernetes and CNCF Finland Meetup
+
+We are proud organizers of the [Kubernetes and CNCF Finland Meetup](https://www.meetup.com/kubernetes-finland/).
+The Meetup was started several years about by [Lucas Käldström](https://www.meetup.com/members/227375788/) later
```
Comment (MaryamTavakkoli) [inline content/en/events/_index.md:25]:
```suggestion
The Meetup was started several years ago by [Lucas Käldström](https://www.meetup.com/members/227375788/) later
```

File: content/en/events/_index.md (hunk 6 above)
Comment (CsatariGergely) [reply content/en/events/_index.md:14]: Do we need the hyphen here?

Comment (MaryamTavakkoli) [reply content/en/events/_index.md:14]: even if not the hyphen, the (us) in between made the reading hard. So, maybe just `community organized event`

File: content/en/_index.md (hunk 1 above)
Comment (CsatariGergely) [reply content/en/_index.md]: Fixed in [3b9a562](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/3b9a5621fba6928fcbb42463b533d9c60ad53a11)

File: content/en/contact/_index.md (hunk 4 above)
Comment (CsatariGergely) [reply content/en/contact/_index.md]: Added in **[3b9a562](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/3b9a5621fba6928fcbb42463b533d9c60ad53a11)**

File: content/en/events/_index.md (hunk 6 above)
Comment (CsatariGergely) [reply content/en/events/_index.md]: Fixed in [2dbfe34](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/2dbfe340356fd982ff6fe40cf504dd51a975eb8f)

**Review by MaryamTavakkoli — APPROVED** [review]

**Pull request merged**